cypress_test/integration_tests/desktop/calc/top_toolbar_spec.js

```diff
@@ -133,6 +133,102 @@ describe('Top toolbar tests.', function() {
 			.should('have.attr', 'style', 'border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000');
 	});
 
+	it('Apply horizontal borders for multiple cells', function() {
+		calcHelper.selectAllMobile();
+		cy.get('#tb_editbar_item_setborderstyle')
+			.click();
+
+		cy.get('.w2ui-tb-image.w2ui-icon.frame09')
+			 .click();
+
+		calcHelper.selectAllMobile();
+
+		cy.get('#copy-paste-container table td')
+			.should(function(cells) {
+				for (var i = 0; i < cells.length; i++) {
+					expect(cells[i]).to.have.attr('style', 'border-top: 1px solid #000000; border-bottom: 1px solid #000000');
+				}
+			});
+	});
+
+	it('Apply horizontal inner borders and vertical outer borders', function() {
+		calcHelper.selectAllMobile();
+		cy.get('#tb_editbar_item_setborderstyle')
+			.click();
+
+		cy.get('.w2ui-tb-image.w2ui-icon.frame10')
+			 .click();
+
+		calcHelper.selectAllMobile();
+
+		cy.get('#copy-paste-container table td')
+			.should(function(cells) {
+				for (var i = 0; i < cells.length; i++) {
+					if (i == 0)
+						expect(cells[i]).to.have.attr('style', 'border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000');
+					else if (i == 1)
+						expect(cells[i]).to.have.attr('style', 'border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000; border-right: 1px solid #000000');
+					else if (i == 2)
+						expect(cells[i]).to.have.attr('style', 'border-top: 1px solid #000000; border-bottom: 1px solid #000000; border-left: 1px solid #000000');
+					else
+						expect(cells[i]).to.have.attr('style', 'border-top: 1px solid #000000; border-bottom: 1px solid #000000');
```

Comment:
If I write here anything as a style value, it will pass. So these assertions do nothing because we have only one cell here.
You should try to understand better what this test code does in the mobile test case and do something similar here too.